File: samples/client/petstore/java/src/main/java/org/openapijsonschematools/client/apis/paths/Usercreatewithlist.java
package org.openapijsonschematools.client.apis.paths;

import org.openapijsonschematools.client.apiclient.ApiClient;
import org.openapijsonschematools.client.configurations.ApiConfiguration;
import org.openapijsonschematools.client.configurations.SchemaConfiguration;
import org.openapijsonschematools.client.paths.usercreatewithlist.Post;

public class Usercreatewithlist extends ApiClient implements
    Post.PostOperation
{
    public Usercreatewithlist(ApiConfiguration apiConfiguration, SchemaConfiguration schemaConfiguration) {
        super(apiConfiguration, schemaConfiguration);
    }
}

File: samples/client/petstore/kotlin/src/main/kotlin/org/openapijsonschematools/client/apis/paths/Usercreatewithlist.kt
package org.openapijsonschematools.client.apis.paths;

import org.openapijsonschematools.client.apiclient.ApiClient;
import org.openapijsonschematools.client.configurations.ApiConfiguration;
import org.openapijsonschematools.client.configurations.SchemaConfiguration;
import org.openapijsonschematools.client.paths.usercreatewithlist.UsercreatewithlistPost;

public class Usercreatewithlist extends ApiClient implements
    UsercreatewithlistPost.PostOperation
{
    public Usercreatewithlist(ApiConfiguration apiConfiguration, SchemaConfiguration schemaConfiguration) {
        super(apiConfiguration, schemaConfiguration);
    }
}